README.md

```diff
@@ -44,6 +44,23 @@ The available development commands are:
 
 Run tests with `npm run test`
 
+### Configuring the app
+
+The app can be configured by modifying:
+- [Ulaanbaatar config](src/config/ulaanbaatar.js)
+- [National config](src/config/national.js)
+
+These files are commented to describe the available options.
+
+**Note:** If you use a new translation key in one of the configuration files, ensure that you add the key to the translation files [found here](src/i18n). See [Translations](#translations) for more info.
+
+### Translations
```

Comment:
Seems fine IMO, it's hard to give more detail without duplicating the Angular Translate docs